bin/compile.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 DIRECTORY=$(cd `dirname $0` && pwd)
-DIST=$DIRECTORY/../dist
+DIST=$DIRECTORY/../dist/optimized
```

Comment:
Can you revert this change?

Reply:
On heroicons’ repo, the new “root” for svg elements is in “optimized folder”, I thought it was ok to reference it as if a nested git repo in local.

Will revert